packages/paste-core/components/inline-code/src/InlineCode.tsx

```diff
@@ -20,7 +20,7 @@ export interface InlineCodeProps extends Partial<Omit<HTMLPasteProps<"div">, "ch
    */
   element?: BoxProps["element"];
   /**
-   * Only the "default" variant should be used as the "minimal" variant is deprecated.
+   * Different inline code block styles. Defaults to "default".
```

Comment:
question: why did you remove the deprecation note

Reply:
didnt mean to check that in :X